src/main/java/edu/kis/powp/commands/OperateToCommand.java

```diff
@@ -0,0 +1,20 @@
+package edu.kis.powp.commands;
+
+import edu.kis.powp.jobs2d.Job2dDriver;
+
+public class OperateToCommand implements DriverCommand{
+    private Job2dDriver job2dDriver;
+    private int x;
+    private int y;
+
+    public OperateToCommand(Job2dDriver job2dDriver, int x, int y) {
```

Comment:
The solution lacks flexibility, using different drivers with commands is difficult (e.g., you could consider setter).